src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBroker.cs

```diff
@@ -213,5 +213,27 @@ private static MsalTokenResponse CreateMsalTokenResponseFromResult(string broker
 
             return MsalTokenResponse.CreateFromBrokerResponse(response);
         }
+
+        public async Task<IEnumerable<IAccount>> GetAccountsAsync(string clientID)
+        {
+            Dictionary<string, string> brokerPayload = new Dictionary<string, string>();
+            brokerPayload.Add(BrokerParameter.ClientId, clientID);
+
+            try
+            {
+                await _brokerHelper.InitiateBrokerHandshakeAsync(_activity).ConfigureAwait(false);
```

Comment:
>_activity [](start = 65, length = 9)

how is this activity getting created correctly if there was a null broker instance created?

Reply:
not a null broker, just a null activity.

Reply:
since this is not an interactive operation no activity is needed